.tekton/collector-component-pipeline.yaml

```diff
@@ -527,6 +527,24 @@ spec:
       operator: in
       values: [ "false" ]
 
+  - name: ecosystem-cert-preflight-checks
```

Comment:
I think, it was placed before `sast-snyk-check` in other pipelines. Let's have it here also that way for consistency (e.g. I sometimes diff pipelines in different repos and it could be inconvenient to see lines moved).

Reply:
fixed in https://github.com/stackrox/collector/pull/1829/commits/e5fdfadcb232881f0f4830f0b2da13f5dd45aeda